### DISCUSSION TITLE
19 November 2024 2300 UTC

### DISCUSSION THREAD

**Discussion by ktoddbrown** [ktoddbrown/SoilDRaH#58] [RegularCalls]
### Agenda

These calls are lead by @ktoddbrown and are generally open calls.
Please contact them for call in information.
The purpose of these meetings are to move forward current issues related to the development of Soil-DRaH.
This may include teaching, mutual accountability, code review, paired programming, or other activities.

## (5 | 2300) Check ins
- Who is here
- Note taking?
- How are you showing up? <https://feelingswheel.com/>

## (15 | 2305) Stuck to unstuck
- *What is everyone working on right now?*
- *What do you need to get unstuck on a topic?*

## (5 | 2340) Backlog
- *What new issues need to be created?*
- *What do we need to talk about in the next 2-4 calls?*

**Comment by graham-louis** [comment]
# Nov 19

## (5 | 2300) Check ins

- Who is here: Graham Louis, Gabriel, Brandon, Dr. Todd-Brown, Maggie, Vaasuki
- Note taking? Graham Louis
- How are you showing up? [https://feelingswheel.com/](https://feelingswheel.com/)

## (15 | 2305) Stuck to unstuck

- _What is everyone working on right now?_
    - _Dr. TB:_ 
        - _ISCN4 Data curation_
        - _CPEAT coded in (Unit extractions from parm descriptions needed_
        - _Needs to continue pulling state names for NRCS_
    - _Graham_
        - _Polishing Longleaf Pine_
        - _Will create data report based on CPEAT report (Create new Issue)_
            - _Histograms for all variables_
            - _Plot Carbon percentage to Bulk density_
            - _Report on Control Vocabulary_
                - _Function called “Table"_
                - _Count Entries_
    - _Brandon_
        - _Reviewing code for CPEAT ingest (still in progress)_
    - _Vaasuki_
        - _Scraping citiations from G Scholar_
        - _Noticed NA vals in Source and Target in col names (in ISCN4map)_
            - _Should this table be created for CPEAT?_
            - _Target_variable are assumed measured_
            - _Target_method is used to distinguish between source_variables that map to the same target_variables_
    - _Gabriel_
        - _Reviewing larger datasets (FIA)_
            - _General How-to documentation_
    - _Maggie_
        - _Working on the_ 
- _What do you need to get unstuck on a topic?_
    - _Maggie_
        - _Include Data provenance in “Find Data”?_
        - _Dr. TB Says this applies more to qa/qc_
        - _In general, in combining data collections we need to check for repeats between collections_
            - _Check for double citations for duplicates? -> CPEAT has citations in related papers_
            - _Start by looking at location and are the values very similar -> Probably a repeat_

## (20 | 2320) Curate ISCN4 walk through [Kathe]

- On method annotations:
    - We care less about measured vs. estimated for our purposes (Mapping and model verification)
    - Target_method should probably be filled out for all variables
    - Could add a column for “source_methodology"
    - Manuscript highlights idea that we make these disperate data sets speak to each other
        - Do we really care about prep codes etc.?
        - Is this level of detail useful for the project’s intended usecase
    - Brandon Asks how do we validate script-based date transforms
        - Histograms provide a certain amount of checking
        - Excel export (day since origin) prevents formatting issues like switching of year and month
- ISCN4 map
    - Describes the map from source variables to the target variables, which are consistent between datasets
- ISCN3_lvl1$Primary
    - Found inconsistencies in upper_depth_bound col - > This should be investigated 
    - Same layer ID’s have different upper and lower bounds

## (5 | 2340) Backlog
*Not discussed*
- _What new issues need to be created?_
- _What do we need to talk about in the next 2-4 calls?_

## (5 | 2345) Check outs [appreciations, discoveries, and ToDo]

- Next call
    - Reflect on semester/ close out
